### PR TITLE
fix: ipfs non default port

### PR DESCRIPTION
**Pull request by jorgeantonio21**

### CODE REVIEW

File: atoma-output-manager/src/lib.rs [hunk 1]
```diff
@@ -51,7 +52,11 @@ impl AtomaOutputManager {
             GatewayOutputManager::new(&config.gateway_api_key, &config.gateway_bearer_token);
 
         info!("Building IPFS client...");
-        let client = IpfsClient::default();
+        let ipfs_host = config.ipfs_host.unwrap_or("localhost");
+        let ipfs_port = config.ipfs_port;
+
+        let client = IpfsClient::from_host_and_port(Scheme::HTTP, ipfs_host, ipfs_port)
```
Comment (Cifko) [inline atoma-output-manager/src/lib.rs:58]:
```suggestion
        let client = IpfsClient::from_host_and_port(Scheme::HTTP, &ipfs_host, ipfs_port)
```

File: atoma-output-manager/src/lib.rs [hunk 2]
```diff
@@ -51,7 +52,11 @@ impl AtomaOutputManager {
             GatewayOutputManager::new(&config.gateway_api_key, &config.gateway_bearer_token);
 
         info!("Building IPFS client...");
-        let client = IpfsClient::default();
+        let ipfs_host = config.ipfs_host.unwrap_or("localhost");
```
Comment (Cifko) [inline atoma-output-manager/src/lib.rs:55]:
```suggestion
        let ipfs_host = config.ipfs_host.unwrap_or("localhost".to_string());
```

File: atoma-input-manager/src/lib.rs [hunk 3]
```diff
@@ -51,16 +55,22 @@ pub struct AtomaInputManager {
...
     ) -> Result<Self, AtomaInputManagerError> {
+        let config = AtomaInputManagerConfig::from_file_path(config_file_path);
+
         info!("Starting Atoma Input Manager...");
         let start = std::time::Instant::now();
         let (ipfs_request_tx, ipfs_request_rx) = mpsc::unbounded_channel();
 
         info!("Building IPFS client...");
-        let client = IpfsClient::default();
+        let ipfs_host = config.ipfs_host.unwrap_or("localhost");
```
Comment (Cifko) [inline atoma-input-manager/src/lib.rs:70]:
```suggestion
        let ipfs_host = config.ipfs_host.unwrap_or("localhost".to_string());
```

File: atoma-input-manager/src/lib.rs [hunk 4]
```diff
@@ -51,16 +55,22 @@ pub struct AtomaInputManager {
...
+        let ipfs_port = config.ipfs_port;
+        let client = IpfsClient::from_host_and_port(Scheme::HTTP, ipfs_port, ipfs_port)
```
Comment (Cifko) [inline atoma-input-manager/src/lib.rs:72]:
```suggestion
        let client = IpfsClient::from_host_and_port(Scheme::HTTP, &ipfs_host, ipfs_port)
```

**Review by Cifko — CHANGES_REQUESTED** [review]
couple issues, but otherwise it works

File: atoma-output-manager/src/lib.rs (hunk 1 above)
Comment (jorgeantonio21) [reply atoma-output-manager/src/lib.rs:58]: Nice catch ! Resolved

**Review by Cifko — APPROVED** [review]

**Pull request merged**